internal/rbd/rbd_util.go

```diff
@@ -792,47 +802,31 @@ func getSnapshotMetadata(ctx context.Context, pSnapOpts *rbdSnapshot, cr *util.C
 	return nil
 }
 
-// imageInfo strongly typed JSON spec for image info
-type imageInfo struct {
-	ObjectUUID string   `json:"name"`
-	Size       int64    `json:"size"`
-	Features   []string `json:"features"`
-	CreatedAt  string   `json:"create_timestamp"`
-}
-
 // getImageInfo queries rbd about the given image and returns its metadata, and returns
 // ErrImageNotFound if provided image is not found
-func getImageInfo(ctx context.Context, monitors string, cr *util.Credentials, poolName, imageName string) (imageInfo, error) {
+func (rv *rbdVolume) getImageInfo() error {
 	// rbd --format=json info [image-spec | snap-spec]
```

Comment:
need to remove this comment as its no more valid

Reply:
of course!